eslint-configs/javascript/index.js

```diff
@@ -1,3 +1,7 @@
 module.exports = {
-  extends: ["eslint:recommended", "plugin:prettier/recommended"]
+  extends: ["eslint:recommended", "plugin:prettier/recommended"],
+  // define parser options to make features such as async/wait work
+  parserOptions: {
+    ecmaVersion: 2020 // same as ECMAScript 10
```

Comment:
I have never seen this (version 2020), but if it works, great ;)

Reply:
same :) but it seems to work